### DISCUSSION TITLE
Is it possible that the webreader automaticaly detects a double page and fits it to height?

### DISCUSSION THREAD

**Discussion by alscaldas** [gotson/komga#1576] [Ideas]
### Describe your suggested feature

Hello. I've just installed the docker version of this project and I must say that I like very much what I've seen so far.

Well, I read my comics mostly in an iPad and since Chunky doesn't sync my reading progress I'm trying to get used to the webreader. In fact it works pretty well, but I'd like to make a suggestion.

These are the options I am using:

Reading mode: Left to Right
Scale type: Fit width (shrink only)
Page layout: Single page

They work perfectly when it comes to comics that have just single pages, but if the comic has any double page (I'm considering 'double page' here any situation in which a page width is larger than its height) the webreader shrinks the page by its width to fit the screen, so that it gets too small to read.

Would it be possible to the webreader, using these configurations mentioned above, automatically detect a double page and only on these pages fit them to height instead of width?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
If you've tried got to height you may have noticed you can't scroll horizontally, so that's probably not a good solution either.

**Comment by zeus163** [comment]
I'd love this as well. I posted something along the lines on the discord site. If you zoom in so that you can read the double page, it is scrollable, but it would be nice if the webreader could detect a double page and git the height. I just wanted to say that I would love this option. Of course, I don't know how to program, so I don't even know if it is possible!
